### DISCUSSION TITLE
所以 到底要怎麼交功課(T T)

### DISCUSSION THREAD

**Discussion by Chengbosco** [1101-GS4719-C/Python-Community-GS4719-C#1905] [General]
請問一下，這樣就算教完功課了嗎?
![Initial commit · NCU-GS4719-Python_1121-python-final-report-Chengbosco@c2e2c07 - Google Chrome 2024_1_16 上午 12_30_27](https://github.com/NCU-GS4719-Python/Python-Community/assets/144880105/8a4b31c9-b536-48b2-855d-2bee11df5ee1)
大一生沒修過這種程式課可所以請問一下(T T)

**Comment by spermwhale464** [comment]
上課老師都有教過，如果沒聽懂也可以私訊老師或助教，其實看起來很難，實則一點都不簡單，我這裡也很難講清出，你還是去問老師吧，不然你真的 要沒分數了
